packages/contentstack-command/package.json

```diff
@@ -17,13 +17,12 @@
     "format": "eslint src/**/*.ts --fix"
   },
   "dependencies": {
-    "@contentstack/cli-utilities": "^1.0.5",
-    "@contentstack/management": "^1.6.0",
-    "@oclif/command": "^1.8.16",
+    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/management": "^1.6.1",
```

Comment:
@antonyagustine , shouldn't we keep oclif core as a dependency

Reply:
**@oclif/core** package will take care of prod dependnc works. So, **oclif** need not to be in dependency.